src/components/Location/Location.jsx

```diff
@@ -0,0 +1,37 @@
+import "./Location.css"
+import { GoogleMap, useLoadScript, MarkerF } from "@react-google-maps/api"
+
+const Location = () => {
+
+    const { isLoaded } = useLoadScript({
+        googleMapsApiKey: import.meta.env.VITE_GOOGLE_API_KEY,
+    });
+
+    if(!isLoaded) return <div><h1>Loading...</h1></div>
```

Comment:
So this works, but if there was an error (which we should be checking for) the loading would forever show. We want our map to be responsive to loading, error or expected states. 

We will update together